lightning/src/ln/blinded_payment_tests.rs

```diff
@@ -54,7 +54,7 @@ fn do_one_hop_blinded_path(success: bool) {
 	nodes[0].node.send_payment(payment_hash, RecipientOnionFields::spontaneous_empty(),
 	PaymentId(payment_hash.0), route_params, Retry::Attempts(0)).unwrap();
 	check_added_monitors(&nodes[0], 1);
-	pass_along_route(&nodes[0], &[&[&nodes[1]]], amt_msat, payment_hash, payment_secret);
+	pass_along_route(&nodes[0], &[&[&nodes[1]]], amt_msat, payment_hash, payment_secret, false);
```

Comment:
The majority of changes to unrelated tests were added in the last commit, where I refactored `pass_along_route` to accommodate for probes. I am not sure whether it would be better to leverage only `do_pass_along_path` (as done in the previous commit 043e6b7bd74b8b487b4bfd0a3c5776865118c1a5) to prevent changing the function signature of `pass_along_route`, which is used in many places.